sdk-extensions/autoconfigure/README.md

```diff
@@ -72,11 +72,18 @@ The OpenTelemetry SDK can be disabled entirely. If disabled, `AutoConfiguredOpen
 
 The following configuration properties are common to all exporters:
 
-| System property       | Environment variable  | Purpose                                                                                                                    |
-|-----------------------|-----------------------|----------------------------------------------------------------------------------------------------------------------------|
-| otel.traces.exporter  | OTEL_TRACES_EXPORTER  | List of exporters to be used for tracing, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
-| otel.metrics.exporter | OTEL_METRICS_EXPORTER | List of exporters to be used for metrics, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
-| otel.logs.exporter    | OTEL_LOGS_EXPORTER    | List of exporters to be used for logging, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter. |
+| System property                             | Environment variable                        | Purpose                                                                                                                                                                                                  |
+|---------------------------------------------|---------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| otel.traces.exporter                        | OTEL_TRACES_EXPORTER                        | List of exporters to be used for tracing, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                               |
+| otel.metrics.exporter                       | OTEL_METRICS_EXPORTER                       | List of exporters to be used for metrics, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                               |
+| otel.logs.exporter                          | OTEL_LOGS_EXPORTER                          | List of exporters to be used for logging, separated by commas. Default is `otlp`. `none` means no autoconfigured exporter.                                                                               |
+| otel.java.experimental.exporter.memory_mode | OTEL_JAVA_EXPERIMENTAL_EXPORTER_MEMORY_MODE | If `reusable_data`, enable reusable memory mode (on exporters which support it) to reduce allocations. Default is `immutable_data`. This option is experimental and subject to change or removal.**[1]** |
+
+**[1]**: NOTE: The exporters which adhere
```

Comment:
Would you like me to add documentation to the website about memory mode?

Reply:
I think there's an issue open somewhere to move all this autoconfigure docs to the website. For the moment, opentelemetry.io just links to this page for anything related to SDK environment variables. I think that's fine for now.

Reply:
I guess I'm asking if we should elaborate more on the memory mode, or the configuration description suffices?